src/client/public/runner.ts

```diff
@@ -126,6 +129,13 @@ interface InitializationParams {
         const $emptySnippetPlaceholder = $('<div class="snippet-frame"></div>')
             .insertAfter($snippetContent);
 
+        if (!isTrustedSnippet) {
+            showReloadNotification($('#notify-snippet-not-trusted'),
+                () => clearAndRefresh(currentSnippet.id, '', true /*isTrustedSnippet*/),
```

Comment:
Why can't we set notifySnippetNotTrustedDisplayed here instead of having to query the DOM?

Reply:
Modified this to not directly query the DOM.